Decred Wallet/Features/More/More Features/Settings/SettingsController.swift

```diff
@@ -209,12 +251,56 @@ class SettingsController: UITableViewController  {
                     StartupPinOrPassword.set(sender: self, done: self.checkStartupSecurity)
                 }
                 
-            case 1: // change startup pin/password
+            case 1: // use biometric
+                self.promptForStartupPinOrPassword() { pinOrPassword, _, dialogDelegate in
+                    self.verifyPinOrPassword(startupPinOrPassword: pinOrPassword, dialogDelegate: dialogDelegate)
+                }
+                
+            case 2: // change startup pin/password
                 StartupPinOrPassword.change(sender: self, done: self.checkStartupSecurity)
                 
             default:
                 break
             }
         }
     }
+    
+    func promptForStartupPinOrPassword(callback: @escaping SecurityCodeRequestCallback) {
+        let prompt = String(format: LocalizedStrings.enableWithStartupCode,
+                            StartupPinOrPassword.currentSecurityType().localizedString)
+        
+        Security.startup()
+            .with(prompt: prompt)
+            .with(submitBtnText: LocalizedStrings.verify)
+            .should(showCancelButton: true)
+            .requestCurrentCode(sender: self, callback: callback)
+    }
+    
+    func verifyPinOrPassword(startupPinOrPassword: String, dialogDelegate: InputDialogDelegate?) {
```

Comment:
Either rename to `verifyStartupPinOrPasswordAndToggleBiometricsUsage` or split this function into 2: `verifyStartupPinOrPassword` and `toggleBiometricsUsage`.

If using 2 functions, the `verifyStartupPinOrPassword` function should receive a `completion: (Bool) -> Void` parameter that will be invoked after verification is done (with true if verified successfully or false if otherwise).
Then when calling that function, you do `self.verifyStartupPinOrPassword(startupPinOrPassword: pinOrPassword, dialogDelegate: dialogDelegate, completion: self.toggleBiometricsUsage)`

Reply:
i just renamed the the function name